node/resolvers/subscriptions/index.ts

```diff
@@ -1,11 +1,25 @@
+import { generateBetweenConstraint, generateOrConstraint } from '../masterDataQueryBuilders'
+
+const SUBSCRIPTION_ORDERS_SCHEMA = "subscription_orders-v1"
+const SUBSCRIPTION_SCHEMA = 'bi-v1'
+
+const fieldResolver = (field) => field.replace(/(_[a-z])/g, char => char.toUpperCase()).replace(/(_)/g, '')
```

Comment:
Seems not a good solution, my propose is create a type resolver for type  `SubscriptionOrdersStatusCount`

check -> [Category type Resolver] https://github.com/vtex-apps/store-graphql/blob/master/node/resolvers/catalog/category.ts#L7
[Export field resolver] https://github.com/vtex-apps/store-graphql/blob/master/node/resolvers/catalog/index.ts#L49
[import inside index]https://github.com/vtex-apps/store-graphql/blob/master/node/resolvers/index.ts#L5
[use]https://github.com/vtex-apps/store-graphql/blob/master/node/resolvers/index.ts#L17